app/main.py

```diff
@@ -1,8 +1,58 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list:
```

Comment:
add additional typehint for return list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]